### PR TITLE
Add possibility to manually rebuild only selected scopes

### PR DESCRIPTION
**Pull request by dkarnutsch**
This should help to reduce load and create faster builds

--------
Design/Implementation considerations:
- mui data grid was used because I did not want to develop the selection manually
- ui/ux was aligned with Mitch
- discussed with @johnnyomair: no mapping from _cronJob to scope_ for the moment. If needed, we will add it later

Screenshot 2022-08-31 at 10 59 51

### CODE REVIEW

File: packages/admin/cms-admin/src/builds/PublishButton.tsx [hunk 1]
```diff
@@ -1,80 +1,30 @@
...
-                    [classes.buttonError]: hasError,
-                })}
-                disabled={loading}
-                onClick={handlePublishClick}
-                fullWidth
-            >
-                {hasStarted && <FormattedMessage id="comet.pages.publisher.buildStarted" defaultMessage="Build started" />}
-                {hasError && <FormattedMessage id="comet.pages.publisher.buildStartFailed" defaultMessage="Starting build failed" />}
-                {!hasStarted && !hasError && <FormattedMessage id="comet.pages.publisher.build" defaultMessage="Start build" />}
+            <Button variant="contained" color="primary" onClick={handlePublishClick} disabled={process.env.NODE_ENV === "development"} fullWidth>
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/builds/PublishButton.tsx:19]: Should be disable the button when a build is already running?

File: packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                <FormattedMessage id="comet.pages.publisher.startBuildsDialog.header" defaultMessage="Start manual builds" />
+            </DialogTitle>
+            <DialogContent>
+                <Alert severity="info">
+                    <FormattedMessage
+                        id="comet.pages.publisher.startBuildsDialog.hint"
+                        defaultMessage="The more builds you start, the longer it takes until they are completed. If you need to rebuild manually, select only the scopes required."
+                    />
+                </Alert>
+                <Box sx={{ height: 400, width: "100%" }}>
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx:58]: Is it possible to have an `auto` height?

**Comment by johnnyomair** [comment]
Is it possible to hide the pagination when we only have one page?

File: packages/admin/cms-admin/src/builds/PublishButton.tsx (hunk 1 above)
Comment (dkarnutsch) [reply packages/admin/cms-admin/src/builds/PublishButton.tsx:19]: I would not do this as you could start a build for scope A and after that you want to start a build for scope B. We could disable the selection for already running builds, but I think that is an overkill, as the api does not trigger a new build if one is already running for that scope

File: packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx (hunk 2 above)
Comment (johnnyomair) [reply packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx:58]: https://mui.com/x/react-data-grid/layout/#auto-height

**Comment by dkarnutsch** [comment]
> Is it possible to hide the pagination when we only have one page?

done

File: packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx (hunk 2 above)
Comment (dkarnutsch) [reply packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx]: auto-height must not be used with autoPageSize. I've had a quick talk with @jamesricky: we decided to go for a fixed page size and auto-height without any surrounding box

**Pull request merged**